src/item.h

```diff
@@ -1364,7 +1367,11 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
         * Returns label from "item_label" itemvar and quantity
         */
         std::string label( unsigned int quantity = 0 ) const;
+
     private:
+        /** unique identifier for each non-null instance */
+        item_uid uid_;
```

Comment:
We aren't using trailing underscores, please don't apply random style elements to the code.
Every item doesn't need a uid, item_location can apply one when it is constructed around an item.